To work around #967954, we'll encourage the use of the version of
flate2 already in debian.

If the buildd network becomes more flexible, or if flate2 1.0.14 gets
included in debian unstable, or if debcargo gets more clever about how
it creates build-dependencies, this should be reverted.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,7 @@
 version = "1.0"
 
 [dependencies.flate2]
-version = ">=1.0.0, <=1.0.14"
+version = ">=1.0.0, <1.0.14"
 optional = true
 default-features = false
